ufc/tests/test-case-kill-switch-flag-2.json

```diff
@@ -0,0 +1,949 @@
+{
+  "flag": "kill-switch-2",
```

Comment:
copy of `test-case-kill-switch-flag.json` but with the flag name changed to use the one with flipped variation order